playbooks/cluster/openshift/config.yml

```diff
@@ -2,31 +2,74 @@
 - hosts: all
   remote_user: root
   pre_tasks:
+    - name: include_vars
+      include_vars:
+        file: "{{ item }}"
+      with_items:
+        - "{{ playbook_dir }}/../../../vars/global_vars.yml"
+        - "{{ playbook_dir }}/vars/{{ openshift_ver }}.yml"
+
+    - name: set SELinux to permissive mode
+      command: setenforce 1
+
+    - name: set SELinux to permissive mode under configuration file
```

Comment:
Can you run this without ```become: true```?

Reply:
You can see that in the top of the playbook that we use `remote_user: root`.
If you think that we should use `become`, we can do it a different PR.